sacn/transmitter.go

```diff
@@ -189,6 +195,14 @@ func (t *Transmitter) sendOut(server *net.UDPConn, universe uint16) {
 	}
 }
 
+func (t *Transmitter) SetKeepAlive(interval time.Duration) {
```

Comment:
Please add a comment preceding the function declaration describing what this function does. I suggest using your commit message.